### DISCUSSION TITLE
After the update, the autocomplete menu changed color

### DISCUSSION THREAD

**Discussion by optic0n** [neoclide/coc.nvim#5018] [Q&A]
How can I get the old colors back?
before update:
![signal-2024-05-21-204408_002](https://github.com/neoclide/coc.nvim/assets/77533678/2c4fde1f-71e5-49a7-b772-64cb03a80f4a)
after update:
![Screenshot_2024-05-21_14-20-14](https://github.com/neoclide/coc.nvim/assets/77533678/9a9afc2a-0d15-4150-8a4d-2cb82498772b)
colorscheme: iceberg
any thoughts?

**Comment by fannheyward** [comment]
What do you mean `After the update`? Upgrading coc.nvim or vim/nvim?

**Reply by optic0n** [reply]
To be honest, I don't know. There was a neovim update, I updated, and then updated all the plugins. And after that, the color of the coc completion menu changed.
